apps/desktop/src/lib/vbranches/types.ts

```diff
@@ -239,15 +239,15 @@ export class DetailedCommit {
 	/**
 	 * The commit ids of the dependencies of this commit.
 	 */
-	commitDependencies!: string[];
+	dependencies!: string[];
 	/**
 	 * The ids of the commits that depend on this commit.
 	 */
-	inverseCommitDependencies!: string[];
+	inverseDependencies!: string[];
```

Comment:
As @krlvi suggested, should we perhaps call this `reverseDependencies`?

Reply:
Right, I'll change it to that 👍